mathics/builtin/exptrig.py

```diff
@@ -1120,3 +1053,27 @@ def apply_xy_dx(self, x, y, dx, dy, steps, evaluation):
         'AnglePath[{{x_, y_}, {dx_, dy_}}, {steps___}]'
         phi0 = Expression('ArcTan', dx, dy)
         return AnglePath._compute(x, y, phi0, steps.get_sequence(), evaluation)
+
+
+class LogisticSigmoid(Builtin):
+    """
+    <dl>
+    <dt>'LogisticSigmoid[$z$]'
+        <dd>returns the logistic sigmoid of $z$.
+    </dl>
+    
+    >> LogisticSigmoid[0.5]
+     = 0.622459
+
+    >> LogisticSigmoid[0.5 + 2.3 I]
+     = 1.06475 + 0.808177 I
+
+    >> LogisticSigmoid[{-0.2, 0.1, 0.3}]
+     = {0.450166, 0.524979, 0.574443}
+    """
+
+    attributes = ('Listable', 'NumericFunction',)
+    
+    rules = {'LogisticSigmoid[z_?NumberQ]': 'Power[Plus[1,Power[E,Times[-1,z]]],-1]'}
```

Comment:
Note: it should be possible to write these rules in regular `InputForm` syntax, i.e. `1 / (1 + Exp[-z])`, which I think is a bit clearer.

Also, I think we should only apply this rule when the argument is real, `LogisticSigmoid[I Pi]` should remain unevaluated (good idea to add this as a test).